routes/auth.js

```diff
@@ -9,9 +9,15 @@ const REDIRECT_URI = process.env.REDIRECT_URI
 const COOKIE_DOMAIN = process.env.COOKIE_DOMAIN
 const AUTH_TOKEN_EXPIRY_MS = process.env.AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS.toString()
 const FRONTEND_URL = process.env.FRONTEND_URL
+const COOKIE_NAME = 'oauthtoken'
```

Comment:
nitpick, but is there a reason the cookie name should not also be in the env file?

Reply:
No reason - just added it in as an env var